tests/benchq/resource_estimation/test_footprint_analysis.py

```diff
@@ -177,50 +185,22 @@ def test_algorithm_failure_prob_calculation():
         num_t=20,
         surface_code_cycle_time=scc_time,
         physical_error_rate=physical_error_rate,
+        hardware_failure_tolerance=1e-1,
     )
     best_cost_t, best_params_t = cost_estimator(
         num_logical_qubits=num_logical_qubits,
         num_toffoli=30,
         num_t=0,
         surface_code_cycle_time=scc_time,
         physical_error_rate=physical_error_rate,
+        hardware_failure_tolerance=1e-1,
     )
     numpy.testing.assert_almost_equal(
         best_cost_toffoli.algorithm_failure_probability,
         best_cost_t.algorithm_failure_probability,
     )
 
 
-def test_default_T_factories():
```

Comment:
What's the reason for removing this test?

Reply:
We used this code to explicitly test that a particular magic state distillation was being used when the error rate what 1e-3. We created this functionality due to a misunderstanding of how the original openfermion code worked. When I returned the code to how openfermion had originally implemented it, this test was no longer necessary.